test/fuzz/fuzzer.ml

```diff
@@ -50,21 +46,32 @@ let check_optimized m =
         "only unoptimized module interpretation throws an error: %s" msg
     | Ok (), Error msg ->
       Format.ksprintf failwith
-        "only optimized module interpretation throws an error: %s" msg
-  in
-  *)
+        "only reference module interpretation throws an error: %s" msg
+  in *)
   result1 && result2
 
 let is_optimized m = Crowbar.check (check_optimized m)
 
+let debug = ref false
+
+let options = [
+  "--debug", Arg.Set debug, " Debug mode (modul printing)"
+]
+
+(* let in_file_name = ref "" *)
+(* let set_file _s = print_endline "FFFFFF" *)
+
+let usage = "usage: dune exec -- ./test/fuzz/fuzzer.exe [--option]"
```

Comment:
```ocaml
let usage = Format.sprintf "usage: dune exec -- %s [--option]" Sys.argv.(0)
```

Reply:
For now, OCaml Arg module could not be used (conflict with crowbar)
So, new hard coded parameters have been added (param.ml)

Reply:
Could you add a TODO to param.ml with a link to https://github.com/OCamlPro/owi/pull/28#discussion_r1231983824 ?